tests/unit/recorders/test_default.py

```diff
@@ -578,11 +579,11 @@ def se():
             mock_publish.reset_mock()
 
             recorder.unpublished_metrics_count = 1
-            frozen_time.tick(9)
+            frozen_time.tick(datetime.timedelta(seconds=9))
```

Comment:
`tick` method expects a delta, not an integer
https://github.com/spulec/freezegun/blob/master/freezegun/api.pyi#L23